draft-ietf-quic-transport.md

```diff
@@ -2241,6 +2258,12 @@ the data that was sent in a previous PATH_CHALLENGE frame.  A PATH_RESPONSE
 frame received on any network path validates the path on which the
 PATH_CHALLENGE was sent.
 
+If the PATH_CHALLENGE frame was sent in a datagram that was not expanded to at
+least 1200 bytes, an endpoint can regard the address as valid for the purposes
+of sending more than three times the amount of data that has been received.
+However, the path MTU is not validated so the endpoint MUST initiate another
+path validation to verify that the path MTU is sufficient.
```

Comment:
If we are to change the MUST above (see my comment above) to "can," then I think we can get rid of this paragraph.